packages/ketcher-polymer-editor-react/src/Editor.tsx

```diff
@@ -32,13 +36,23 @@ function Editor(props: EditorProps) {
     onInit?.()
   }, [onInit])
 
+  const RootContainer = styled.div({
+    height: '100%',
+    width: '100%',
+    position: 'relative',
+    minWidth: 640,
+    minHeight: 400,
+    boxSizing: 'border-box'
+  })
+
   return (
-    <div
-      ref={rootElRef}
-      className={clsx('Ketcher-polymer-editor-root', classes.root)}
-    >
-      <AppContainer />
-    </div>
+    <Provider store={store}>
+      <ThemeProvider theme={theme}>
```

Comment:
There's also an interesting hack @AnaMoskvina used in Miew in [this PR](https://github.com/epam/miew/pull/368/files#diff-b005246af4f160c36f839dc96e255eea1f7424d3c9594e3a54719b7f4b8ea1ee).

If it works well, we can merge theme providers for emotion and MUI. We might need MUI theme to [disable ripple effects globally](https://mui.com/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally), or do other adjustments for MUI's defaults.

Reply:
It's already merged into Mui theme via createTheme https://mui.com/customization/theming/#custom-variables